src/test/kotlin/uk/gov/dluhc/printapi/jobs/InitialRetentionPeriodDataRemovalJobIntegrationTest.kt

```diff
@@ -29,6 +33,10 @@ internal class InitialRetentionPeriodDataRemovalJobIntegrationTest : Integration
         assertThat(certificateRepository.findById(certificate2.id!!).get()).initialRetentionPeriodDataIsRemoved()
         assertThat(certificateRepository.findById(certificate3.id!!).get()).hasInitialRetentionPeriodData()
         assertThat(certificateRepository.findById(certificate4.id!!).get()).hasInitialRetentionPeriodData()
+        assertThat(deliveryRepository.findById(deliveryId1)).isEmpty
+        assertThat(deliveryRepository.findById(deliveryId2)).isEmpty
+        assertThat(deliveryRepository.findById(deliveryId3)).isNotEmpty
+        assertThat(deliveryRepository.findById(deliveryId4)).isNotEmpty
```

Comment:
As an improvement, separate to this, I'll look into adding a Repository class for the `Address` entity within this integration test, so that I can confirm it's been deleted. I've tested it manually for now.